File: Programmers/Inho/C#/Level01/핸드폰 번호 가리기.cs
using System;
public class Solution {
    public string solution(string phone_number) {
        string answer = "";
        answer = phone_number;
        string temp = "";
        for(int i=0;i<phone_number.Length-4;i++){
            temp +="*";
        }
        for(int i = phone_number.Length-4;i<phone_number.Length ; i++){
            temp +=phone_number[i];
        }
        answer = temp;
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/핸드폰 번호 가리기.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

string solution(string phone_number) {
    string answer = "";
    for(int i =0;i<phone_number.length()-4;i++){
        answer+='*';
    }
    answer += phone_number.substr(phone_number.length()-4);
    
    return answer;
}